charts/em-icp/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+name: em-icp
+home: https://github.com/hmcts/rpa-em-icp-api
```

Comment:
Do we need to move the github repo as we did for others? i.e. From rpa-em-icp-api to em-icp-api ?